clar2wasm/src/tools.rs

```diff
@@ -366,7 +366,7 @@ impl TestEnvironment {
 
 impl Default for TestEnvironment {
     fn default() -> Self {
-        Self::new(StacksEpochId::Epoch25, ClarityVersion::Clarity2)
+        Self::new(StacksEpochId::latest(), ClarityVersion::latest())
```

Comment:
I don't think this is a good idea. This could make the updating of the stacks-core dependencies a real pain if `latest()` changes and introduces an issue. We have ZERO control on when those value will change. All we can do in this case is stopping to update clarity and stacks-common.
In my opinion, changing the default test environment should always be to a fixed version in dedicated PRs that we control.

Reply:
That makes sense. Will reflect that on the PR.